README.md

```diff
@@ -1,30 +1,48 @@
+- Give an overview of the official guidelines for [naming](/guidelines/naming.md) objects within Kyma
+- Collect [architecture decision records](/architecture-decision-records)
+- Summarize the [content strategy](guidelines/content-guidelines/content-strategy.md) for Kyma  
+- Describe the [release process](/guidelines/releases) in Kyma
+
+### Communication
+
+Contact the Kyma community using various communication channels.
 
-- Host the Kyma [Manifesto](https://kyma-project.github.io/community/).
-- Outline general [contributing rules](CONTRIBUTING.md) for all repositories in Kyma.
-- Specify the [governance](governance.md) policy, the role of a maintainer, and the process of becoming a maintainer.
-- Describe the recommended [Git workflow](git-workflow.md) that relies on forks, branches, rebasing, and squashing.
-- Collect [architecture decision records](/architecture-decision-records).
-- Give an overview of the official guidelines for [naming](/guidelines/naming.md) objects within Kyma.
-- Collect information on public [Special Interest Groups and Working Groups](sig-and-wg/README.md) in Kyma.
-- Define [content guidelines](/guidelines/content-guidelines) for creating diagrams, style, and formatting of the Kyma content.
-- Summarize the [content strategy](guidelines/content-guidelines/content-strategy.md) for Kyma.
-- Describe the [release process](/guidelines/releases) in Kyma.
-- Provide [templates](/guidelines/templates) for the project-related documents.
-- Define [internal guidelines](/guidelines/internal-guidelines) that apply only to the Kyma organization members. These contain a folder, file, and document structure required for a [skeleton Kyma repository](/guidelines/internal-guidelines/repository-template).
+- To ask a **general query**, write to the community on [Slack](https://kyma-community.slack.com/messages/CBLBESMST/). You can also join the regular [SIG Core meetings](https://kyma-community.slack.com/messages/CBP7LKRPS/) that serve as a communication bridge between Kyma representatives and external contributors.
 
-Follow the `community` repository rules and ensure that you contribute high-quality code and content to your project.
+> **NOTE:** Read about [SIG Core](https://github.com/kyma-project/community/blob/master/sig-and-wg/sig-core/README.md) responsibilities and the concepts of [Special Interest Groups and Working Groups](sig-and-wg/README.md) in Kyma.
 
-### Reference
+- To request a **feature** or report a **bug**, raise an issue in a given repository using the templates available.
+Owners of the given repository will review your issue and provide you with their feedback. If you are not certain which repository your issue refers to, open it in the `kyma` repository. Read more about the process in the [CONTRIBUTING.md](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#report-an-issue) document.
```

Comment:
`The owners` ?

Reply:
I rephrased it a bit not to repeat the word "given".